File-Formats/DocIO/Create-Word-document-in-Console-application.md

```diff
@@ -0,0 +1,533 @@
+---
+title: Create Word document in Console Application | Syncfusion 
+description: Create a Word document in Console Application using .NET Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Create a Word document in Console Application
+
+Syncfusion DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **create a Word document in Console Application**.
+
+## Create a Word document using .NET Core and Latest
+
+The below steps illustrates creating a simple **Word document** in console application using **.NET Core**.
+
+Step 1: Create a new **Console application** project.
```

Comment:
Mention what type of console (.NET Framework or .NET Core console).
Add all the screenshots of Visual Studio to identify what framework you choosing for project

Reply:
Modified